src/typescript.ts

```diff
@@ -15,6 +15,17 @@ function nameIsReservedKeyword(name: string): boolean {
   return reservedKeywords.indexOf(name) !== -1;
 }
 
+/**
+ * Will determine whether the tableName should be prefixed with the schemaName
+ */
+export function getEnhancedTableName(
```

Comment:
I don't love "enhanced" -- maybe `getTableTypeName`?

Reply:
Yep I like that better